styles.css

```diff
@@ -0,0 +1,510 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+/* Variables */
+root {
+  --color-shadow: rgb(59 10 39);
+}
+
+body {
+  font-size: 100%;
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Lato', sans-serif;
+}
+
+/* HEADER */
+.bg-home {
+  background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0.3), rgba(255, 255, 255, 0.6)), url(Images/Home/Imagen-copa-tarde.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+header {
+  height: 3em;
+  background-color: transparent;
+  margin: 1rem;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the `background-colour`, `font-size` and `header` spacing correctly. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/b605f815-9188-4238-b5df-f1af9224e04c)


### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-22 16-22-42](https://github.com/ulises2607/cayafate-project/assets/66476329/cd4f1a37-f1a3-4327-ac45-744b0b33378a)

</td>
<td>

![Screenshot from 2023-07-08 15-31-48](https://github.com/ulises2607/cayafate-project/assets/66476329/241e108a-0382-44f0-add2-31d995fd5512)


</td>
</tr>
</table>